frontend/src/libraries/explorer-wamp/transactions.ts

```diff
@@ -57,12 +57,17 @@ export interface Action {
   DeleteKey: DeleteKey;
 }
 
+export interface ActionWrapper {
```

Comment:
Since we are going to use this interface down the line, I think it is fair to rename it to `Action` and the `Action` we have above rename to `RpcAction`.